changelog.d/7530.bugfix

```diff
@@ -0,0 +1 @@
+The green dot for unread rooms was also showing for silent rooms
```

Comment:
You should use [keyword issue linking](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) in your comit message if you have a ticket number e.g. `Fixes #7350 - Fix green dot only to appear for marked action`. 

Also I would squash merge this because of the extra `added changelog` commit, that doesn't bring any value history wise.

P.S. Try to be consistent with your verbs, imperative is the way to go e.g. `add` vs. `added`, `fix` vs. `fixed`

Reply:
Thanks, I'll squash like you suggest